lib/infusion/Version.md

```diff
@@ -0,0 +1,33 @@
+This version of Infusion was created using a custom build from the master branch:
+
+    https://github.com/fluid-project/infusion/
+    commit 18939c0dd614901bfc434a29f880aa332600a239
+
+using the command line:
+    grunt custom --source=true --include="uiOptions" --name="uio"
```

Comment:
Could you modify the format a bit? Remove the indents on the url and commit number so they don't show up like code. For the actual command line code, it should have an empty line above and below, and optionally the three `. You can see how it looks in the first discovery tool. 
https://raw.githubusercontent.com/fluid-project/first-discovery/master/src/lib/infusion/VERSION.md